openquake/mbt/tools/model_building/dclustering.py

```diff
@@ -78,6 +78,8 @@ def decluster(catalogue_hmtk_fname, declustering_meth, declustering_params,
         It can be a string or a list of strings
     :param str tr_fname:
         An .hdf5 file containing the TR classification of the catalogue
+    :param array mag_perm:
```

Comment:
I'm guessing we no longer want a mag_perm input, since it doesn't appear anywhere else